### PR TITLE
Documentation changes to resolve pyne testing issues 

### PR DESCRIPTION
**Pull request by skc222**
Changes to resolve https://github.com/pyne/pyne/issues/896 and https://github.com/pyne/pyne/issues/880
@katyhuff, please review and let me know if something is wrong

### CODE REVIEW

File: docs/install/linux_source.rst [hunk 1]
```diff
@@ -12,8 +12,9 @@ PyNE has the following dependencies:
    #. `Fortran compiler <https://gcc.gnu.org/wiki/GFortran>`_
    #. `C++ compiler <https://gcc.gnu.org/>`_
    #. `CMake <http://www.cmake.org/>`_ (>= 2.8.5)
-   #. `NumPy <http://www.numpy.org/>`_ (>= 1.8.0)
+   #. `NumPy <http://www.numpy.org/>`_ (1.11.0)
```
Comment (scopatz) [inline docs/install/linux_source.rst:15]: This should be >= 1.11

File: docs/install/linux_source.rst [hunk 2]
```diff
@@ -12,8 +12,9 @@ PyNE has the following dependencies:
...
    #. `SciPy <http://www.scipy.org/>`_
+   #. `Jinja2 <http://jinja.pocoo.org/docs/2.9/intro/>`_
```
Comment (scopatz) [inline docs/install/linux_source.rst]: Where is Jinja needed? And is it optional?

**Comment by scopatz** [comment]
Thanks for the updates @skc222!

File: docs/install/linux_source.rst (hunk 1 above)
Comment (skc222) [reply docs/install/linux_source.rst:15]: version >1.11 gives error for test_endf.py 
Should I still make it  >=1.11?

Comment (scopatz) [reply docs/install/linux_source.rst:15]: hmm then it is really not correct to say it is dependent on 1.11 :)

File: docs/install/linux_source.rst (hunk 2 above)
Comment (skc222) [reply docs/install/linux_source.rst]: Actually some tests were giving errors before jinja2 installation
I am not sure if it is optional though. Should I add it in optional dependencies?

Comment (scopatz) [reply docs/install/linux_source.rst]: We should be careful not to overlist dependencies.  Can you hunt down exactly what needs jinja please?

Comment (skc222) [reply docs/install/linux_source.rst]: Yes sure! I will check

Comment (skc222) [reply docs/install/linux_source.rst]: test_decay.py gives error as it imports decaygen which then imports jinja2
```
Ran 70 tests in 1.013s

OK


Testing test_decay.py:

E
======================================================================
ERROR: Failure: ImportError (No module named jinja2)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/ubuntu/Downloads/pyne-0.5.3/tests/test_decay.py", line 33, in <module>
    import decaygen
  File "/home/ubuntu/Downloads/pyne-0.5.3/src/decaygen.py", line 12, in <module>
    import jinja2
ImportError: No module named jinja2

----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (errors=1)
Testing failed on test_decay.py
ubuntu@ubuntu:~/Downloads/pyne-0.5.3/tests$ 
```

**Review by skc222 — COMMENTED** [review]
Please let me know if some other changes are required

File: docs/install/linux_source.rst (hunk 2 above)
Comment (scopatz) [reply docs/install/linux_source.rst]: Please say that jinja is needed only for testing, then. Thanks!

Comment (skc222) [reply docs/install/linux_source.rst]: Yes, I removed jinja2 from dependencies and added it in testing instructions in latest commit

**Comment by scopatz** [comment]
Ok Thanks!

**Pull request merged**